public/components/views/home/maintainers/maintainers.js

```diff
@@ -26,14 +26,24 @@ export class Maintainers {
   }
 
   render() {
-    const authors = [...this.secureDataSet.authors.entries()]
-      .sort((left, right) => right[1].packages.size - left[1].packages.size);
+    const authors = this.#highlightContacts([...this.secureDataSet.authors.entries()]
+      .sort((left, right) => right[1].packages.size - left[1].packages.size));
 
     document.getElementById("authors-count").innerHTML = authors.length;
     document.querySelector(".home--maintainers")
       .appendChild(this.generate(authors));
   }
```

Comment:
I think we need to apply a special CSS (to make sure to properly spot highlighted in the UI)